testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/tracing/TracingTestBase.java

```diff
@@ -120,17 +120,20 @@ protected void putTestEvents(Map<String, Boolean> results) {
    protected void verifyResults(Response response, Map<String, Boolean> results) {
       for (Map.Entry entry : response.getStringHeaders().entrySet()) {
          LOG.info("<K, V> ->" + entry);
-         String item = entry
-               .getValue()
-               .toString()
-               .split("\\[")[1].split(" ")[0];
-
-         if (results.keySet()
-               .contains(item)) {
-            results.put(item.replaceAll(" ", ""), true);
+
+         try {
+            String item = entry
+                    .getValue()
+                    .toString()
+                    .split("\\[")[1].split(" ")[1];
+
+            if (results.keySet()
+                    .contains(item)) {
+               results.put(item.replaceAll(" ", ""), true);
+            }
+         } catch (ArrayIndexOutOfBoundsException e) {
+            LOG.info("[BYPASSED] -> " + entry);
```

Comment:
Is this right? or perhaps the test should fail?

Reply:
@asoldano it's right, because there is irrelevant information in response. maybe the 'BYPASSED' is misleading log, let me change it to 'irrelevant'.